### ISSUE TITLE
glances-version.db Permission denied

### ISSUE THREAD

**Issue by nicolargo** [nicolargo/glances#1066]
#### Description

I just tried with python2.7 and I do have some stacktrace while its running, but glances keeps running (doesn't crash like in python3.6). Gives something a bit ugly like this:

![image](https://cloud.githubusercontent.com/assets/1264761/24335367/7cf3a7d0-1249-11e7-9090-f0c0822861d8.png)

Here's in plain text (I cleaned output a little):

```
Exception in thread Thread-7:
Traceback (most recent call last):
File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 801, in __bootstrap_inner
self.run()
File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 754, in run
self.__target(*self, **self.__kwargs)
File "/Users/gableroux/.virtualenvs/glances27/lib/python2.7/site-packages/glances/outdated.py", line 163, in _update_pypi_version
self._save_cache()
File "/Users/gableroux/.virtualenvs/glances27/lib/python2.7/site-packages/glances/outdated.py", line 143, in _save_cache
with open(self.cache_file, 'wb') as f:
IOError: [Errno 13] Permission denied: '/Users/gableroux/Library/Caches/glances/glances-version.db'
```

in case it's useful:

```
ls -la /Users/gableroux/Library/Caches/glances/glances-version.db
-rw-r--r--  1 root  staff  137 24 Mar 15:59 /Users/gableroux/Library/Caches/glances/glances-version.db
```

#### Versions

* Glances (glances -V): 2.8.8
* PSutil (glances -V): 5.2.1
* Operating System (lsb_release -a): MacOS

**Comment by nicolargo** [comment]
Corrected in the latest commit (4825f2e) in the DEVELOP branch.

Will be release in Glances 2.9.

**Comment by GabLeRoux** [comment]
Thanks for the fix. Also, deleting the file manually and running the `glances` again did fix the problem by setting the right owner on the file. 👍
